wiki/People/Beatmap_Nominators/General_Information/en.md

```diff
@@ -1,57 +1,49 @@
-
-Lastly, the information you will probably be referencing the most, the Ranking Criteria. If you are already a Beatmap Nominator, then you should be familiar with this. Essentially, it serves to explain what aspects of mapping are and are not allowed in the Ranked section, as well as including some general guidelines that people should try to follow in most cases. If you see a Qualified map breaking one of the rules, or feel that it could still use substantial improvement for whatever reason, feel free to report it in this thread, giving a brief outline of the problems you see.
+Are you a Beatmap Nominator, or are interested in becoming one? Then you have come to the right place! This article is all the general information you will need to know as a new Beatmap Nominator. To start, this article will start with a few useful links and then introduce you to the way icons are used, as well as how this transitions into [Beatmap Discussions](/wiki/Beatmap_Discussions) (also known as Modding v2 or Modding Discussions).
 
-For beatmaps with subjective quality concerns that don't necessarily break the rules and guidelines of the Ranking Criteria, but which you feel which make the beatmap in question unfit for the Ranked section, the [_beatmap veto_](/wiki/People/Beatmap_Nominators/Beatmap_Veto) allows you to withhold a beatmap from Qualification.
+## Important Links
 
-## Beatmap Nominator Icons
+- [Beatmap Nominator Rules](/wiki/Beatmap_Nominator_Rules) - which outlines both how to undertake various actions such as iconing maps, as well as how not to do things - you would want to avoid breaking these in any situation, so be sure to read them carefully! You can always ask a member of the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) if you are unsure about something, and we will be happy to help.
 
-Icons are the evidence of a Beatmap Nominator's efforts when checking maps, and though there are many on offer to you, these are the relevant ones to take note of:
+- [Code of Conduct for Mapping and Modding](/wiki/CoC) - As a Beatmap Nominator, you are expected to be an exemplary figure of the community, someone others look up to and respect, so misconduct may get you reprimanded or even expelled.
 
-### ![bubble icon](/wiki/shared/icon/bubble.gif) Bubble
+- [Ranking Criteria](/wiki/Ranking_Criteria) - If you are already a Beatmap Nominator, then you should be familiar with this. Essentially, it serves to explain what aspects of mapping are and are not allowed in the Ranked section, as well as including some general guidelines that people should try to follow in most cases. If you see a Qualified map breaking one of the rules, or feel that it could still use substantial improvement for whatever reason, feel free to report it in [this thread](https://osu.ppy.sh/community/forums/topics/447428), giving a brief outline of the problems you see.
```

Comment:
seems outdated.

Reply:
I wish the qualified beatmap thread was outdated but we don't have a better solution yet. But will seperate into its own item.